src/query/graphite/common/_blacklistgo

```diff
@@ -0,0 +1,229 @@
+package common
```

Comment:
Same with this one.

Reply:
Oops forgot to delete the ones in /common/